tests/OmniSharp.Roslyn.CSharp.Tests/SignatureHelpFacts.cs

```diff
@@ -226,6 +226,36 @@ public MyTestAttribute(int value1,double value2)
         }
 
         [Fact]
+        public async Task SignatureHelpforAttCtorNoParam()
+        {
+            const string source =
+@"using System;
+[MyTest($$)]
+public class TestClass 
+{
+    public static void Main()
+    {
+    }
+}
+public class MyTestAttribute : Attribute 
+{
+    int value1;
+    double value2;
+    public MyTestAttribute()
+    {
+        this.value1 = 1;
```

Comment:
You can delete all the "value1" and "value2" related stuff since this test doesn't use them.